src/utils.ts

```diff
@@ -237,3 +237,44 @@ export function isReviewPending(review: UserReviewData) {
     review.archivedAt === null
   );
 }
+
+// Convert a string to an array
+export const convertStringToArray = (text: string[] | string) => {
+  let stringArray = text as string;
+  if (stringArray[0] === "[") {
+    // eslint-disable-next-line prettier/prettier
+    let _parsed = stringArray
```

Comment:
no need to prefix the variable with an underscore. Also, if we ain't re-assigning the variable, lets make it a `const`